MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceViewQtGUI.cpp

```diff
@@ -1276,14 +1285,7 @@ void TomographyIfaceViewQtGUI::flatsPathCheckStatusChanged(int status) {
 
 void TomographyIfaceViewQtGUI::darksPathCheckStatusChanged(int status) {
   bool enable = 0 != status;
```

Comment:
`bool enable (0 != status);` is much nicer to read. Additionally what does 0 represent in this case?